runtime/rasdump/dmpsup.c

```diff
@@ -577,7 +578,27 @@ configureDumpAgents(J9JavaVM *vm)
 		}
 		if ( IS_CONSUMABLE(j9vm_args, xdumpIndex) && !IS_CONSUMED(j9vm_args, xdumpIndex) )
 		{
-			GET_OPTION_VALUE(xdumpIndex, ':', &optionString);
+			/* Handle mapped tool dump options */
+			if (HAS_MAPPING(j9vm_args, xdumpIndex)) {
+				char *toolCursor = strstr(MAPPING_J9NAME(j9vm_args, xdumpIndex), ":tool") + 1;
+				if (toolCursor) {
+					/* The mapped option specifies the tool command to run after the equals */
+					char *optionValue = NULL;
+					GET_OPTION_VALUE(xdumpIndex, '=', &optionValue);
+					
+					/* Construct optionString by combining the J9 tool dump command with the mapped option */
+					int toolCursorLength = strlen(toolCursor);
+					int optionValueLength = strlen(optionValue);
+					optionString = (char *) j9mem_allocate_memory(toolCursorLength + optionValueLength + 1, OMRMEM_CATEGORY_VM);
+					strcpy(optionString, toolCursor);
+					strncat(optionString + toolCursorLength, optionValue, optionValueLength);
```

Comment:
- Please check for memory allocation failure and handle is the same as other places.
- You don't need `strncat` since you know that `optionValue` will fit.